src/pages/index.js

```diff
@@ -88,46 +118,28 @@ function handleAddCardFormSubmit(values) {
     name: values.title,
     link: values.link,
   };
-  addCard(newCard);
-
-  // closeModal(addCardModal);
-  addCardPopup.close();
-  formValidators["card-form"].disableButton();
-  // formValidators["card-form"].resetValidation();
-  addCardPopup.getForm().reset();
+  // renderLoading true
+  renderLoading("#add-card-modal", true);
+  api
+    .createACard(newCard)
+    .then((cardData) => {
+      addCard(cardData);
+      console.log(cardData);
+      addCardPopup.close();
+      formValidators["card-form"].disableButton();
+      addCardPopup.getForm().reset();
+      //render Loading back to false
+      renderLoading("#add-card-modal", false);
```

Comment:
`      renderLoading("#add-card-modal", false);
`

You need to **only** return the **default** button text in the `finally` block, so that the button could get the text regardless of whether the request succeeds or fails. 


**Please, correct it everywhere in the code**